staging/com.scoreSmart/device.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+const Tp = require('thingpedia');
+const unirest = require("unirest");
+
+module.exports = class ScoreDevice extends Tp.BaseDevice {
+  constructor(engine, state) {
+    super(engine, state);
+    this.uniqueId = 'com.scoreSmart';
+    this.name = 'ScoreSmart';
+    this.description = 'Keep up with the latest in soccer';
+  }
+
+  // getting league standings, given league id
+  get_standings({ league_id }) {
+    const request = require('request');
+
+    const options = {
+      method: 'GET',
+      url: 'https://api-football-v1.p.rapidapi.com/v3/standings',
+      qs: { season: '2020', league: league_id },
+      headers: {
+        'x-rapidapi-key': 'b9b9b1e851msh45c99175d8f8df8p103e18jsn903a05143134',
```

Comment:
Don't put the key in a public repository, or someone can steal it! Instead, create a secrets.json file containing:
```
{"api_key":"..."}
```
And then declare in the manifest:
```
import config from @org.thingpedia.config.none(api_key=$?);
```

You can then access the api_key as `this.constructor.metadata.auth.api_key`
(look at other devices for examples)